spec/index.bs

```diff
@@ -6682,6 +6726,61 @@ interface mixin GPUProgrammablePassEncoder {
     Otherwise return `true`.
 </div>
 
+<div algorithm>
```

Comment:
Didn't the group to make this not a validation error for now and revisit if it proves to cause issues? This PR does the opposite right now.

> Resolution:
> For now, add something to the spec that says "bindings must not alias", but don't add an actual validation error. Add an "Issue(#1842):" that says to revisit this and figure out if we should add a validation error.

I'm also pretty concerned by the additional validation cost that this implies. We are no longer looking at usages for sync scopes only but on every draw?

Reply:
"bindings must not alias" is not a specification
"algorithm: if bindings alias do XXX" is a specification
What do you suggest XXX to be? Say it's undefined?

Reply:
Yeah that's what the group agreed to: "it's bad to do this, don't do it" but not validate it.

Reply:
That's what we agreed to: say it is UB and that people should not do it. For draws you're not avoiding any UB anyway because there is more than one pipeline executing concurrently in the sync scope.

Reply:
I agree this was what we resolved to do but Myles and now Dzmitry object to it, so I guess we have to revisit it.

Reply:
I'm still totally for the spirit of our resolution. But the exact wording is tricky... I don't think the spec can say "this is UB", that's all.

Reply:
In the editors meeting Dzmitry proposed to just say "Issue: figure out what happens if the application violates this restriction." I think this is good; it correctly reflects the fact that we simply haven't decided.